kedro-datasets/RELEASE.md

```diff
@@ -4,6 +4,13 @@
 
 ## Bug fixes and other changes
 
+# Release 1.2.0:
+
+## Bug fixes and other changes
+* Added `fsspec` resolution in `SparkDataSet` to support more filesystems.
+* Added the `_preview` method to the Pandas `ExcelDataSet` and `CSVDataSet` classes.
```

Comment:
This should go under `## Major features and improvements:`

Reply:
I think actually only "Fixed a docstring in the Pandas `SQLQueryDataSet` as part of the Sphinx revamp on Kedro." should go under bug fixes and other changes